.pipelines/ci.yml

```diff
@@ -36,167 +36,125 @@ variables:
-
-            # Install podman
-            sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
-            sudo dnf install -y podman make
-            which podman
-            podman --version
-            whoami
-
-            # Start podman service
-            . ./hack/e2e/utils.sh
-            run_podman
-            validate_podman_running
-
-            # run build
-            export PODMAN_REMOTE_ARGS="-r --url=tcp://localhost:8888"
-            make ci-azext-aro
-
-            # Stop podman
-            kill_podman
-          target: ubi8
-
-    - job: Lint_Az_ARO_Extension
-      pool:
-        name: 1es-aro-ci-pool
-      variables:
-        HOME: $(Agent.BuildDirectory)
-      steps:
-        - template: ./templates/template-checkout.yml
-        - script: |
-            set -xe
-            export AZDEV_CONFIG_DIR=$(Agent.BuildDirectory)/azdev-config
-            make test-python
-            [[ -z "$(git status -s)" ]]
-          target: python
+  - stage: NotContainerized
+    dependsOn: Containerized
+    jobs:
+      - job: Python_Unit_Tests
+        pool:
+          name: 1es-aro-ci-pool
+        variables:
+          HOME: $(Agent.BuildDirectory)
+
+        steps:
+          - template: ./templates/template-checkout.yml
+          - script: |
+              set -xe
+              make test-python
+              [[ -z "$(git status -s)" ]]
+            displayName: 🧪Run Python Unit Tests
+            target: python
+
+      - job: Golang_Unit_Tests
+        pool:
+          name: 1es-aro-ci-pool
+        variables:
+          GOCACHE: /tmp/gocache
+        steps:
+          - template: ./templates/template-checkout.yml
+
+          - script: |
+              set -xe
+              go version
+              go env
+            displayName: Print Go version & env
+            target: golang
+
+          - script: |
+              echo "##vso[task.prependpath]$(go env GOPATH)/bin"
+            displayName: Add GOBIN to path
+            target: golang
+
+          - script: |
+              set -xe
+              # Required for podman 5
+              sudo tdnf install -y gpgme-devel lvm2-devel btrfs-progs-devel golang-1.21.11-1.cm2
+              make generate
+              [[ -z "$(git status -s)" ]]
+            displayName: ⚙️ Run Golang code generate
+            target: golang
+
+          - script: |
+              set -xe
+              make build-all
+              [[ -z "$(git status -s)" ]]
+            displayName: 🕵️ Build Golang code
+            target: golang
+
+          - script: |
+              set -xe
+              make unit-test-go
+            displayName: 🧪 Run Golang unit tests
+            target: golang
+
+          - task: PublishTestResults@2
+            displayName: 📊 Publish tests results
+            inputs:
+              testResultsFiles: $(System.DefaultWorkingDirectory)/**/report.xml
+            condition: succeededOrFailed()
```

Comment:
while we're here - let's fix a quick bug - this step needs to run in `target: golang`

Reply:
Added target: golang to the PublishTestResults step.